README.md

```diff
@@ -29,10 +29,10 @@ The lighting pattern is defined as follows.
 
 ## Input and Output
 - input
-  - from eve oss
+  - from [autoware_state_machine](https://github.com/eve-autonomy/autoware_state_machine)
     - `/autoware_state_machine/lock_state` : Reservation status for on-demand delivery.
 - output
-  - to tier iv oss
+  - to [dio_ros_driver](https://github.com/tier4/dio_ros_driver)
     - `/dio/dout3` : GPIO output topic. (this topic remapping from /delivery_reservation_lamp_out)
```

Comment:
```suggestion
    - `/dio/dout3` : Digital-out assignment to a 3rd pin in [0-7] general-purpose outputs. This topic is remapped from `/delivery_reservation_lamp_out`.
```

And, add a description about values true/false, 1/0, against to the output from the /dio_ros_driver node.